source/nodejs/adaptivecards-site/pages/_posts/2020/Updated-Versioning-Policy.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Updated Versioning and Release policy
+subtitle: Packaging and release software can be complicated, here's what we're doing
+date: 2020-07-05
+github_username: matthidinger
+twitter: matthidinger
+---
+
+## THIS IS A DRAFT
+
+This is an early proposal and we may change these decisions before publishing this post :)
+
+# Updated Versioning and Release policy
+
+Versioning, releasing, and marketing software can be complicated. We're making some changes this year to clarify a few things that should help our customers and developers better understand **what the versions mean**, and **when to expect updates**.
+
+## TL;DR
+
+- Adaptive Cards is an umbrella term for multiple interconnected products: the Adaptive Card schema, the renderer SDKs, the templating schema and templating SDKs, and the card designer.
+- Each of those products exist on different platforms and have their own release cycles.
```

Comment:
Is this still accurate at this point? With the monthly releases we are trying to consolidate release **cycles** . Perhaps use have their own shipping artifacts with versions decoupled from the other.